frontend/src/libs/components/app/app.tsx

```diff
@@ -7,19 +7,45 @@ import {
   useAppSelector,
   useEffect,
   useLocation,
+  useMemo,
+  useNavigate,
 } from '~/libs/hooks/hooks.js';
 import { actions as userActions } from '~/slices/users/users.js';
 
+import { iconNameToSvg } from '../icon/maps/maps.js';
+
 const App: React.FC = () => {
   const { pathname } = useLocation();
   const dispatch = useAppDispatch();
+  const navigate = useNavigate();
   const { users, dataStatus } = useAppSelector(({ users }) => ({
     users: users.users,
     dataStatus: users.dataStatus,
   }));
 
   const isRoot = pathname === AppRoute.ROOT;
 
+  const menuItems = useMemo(
+    () => [
+      {
+        label: 'View history',
+        onClick: () => navigate(AppRoute.ORDER_HISTORY),
+        icon: iconNameToSvg['clock rotate left'],
+      },
+      {
+        label: 'Edit profile',
+        onClick: () => navigate(AppRoute.EDIT_PROFILE),
+        icon: iconNameToSvg['user pen'],
+      },
+      {
+        label: 'Log out',
+        onClick: () => navigate(AppRoute.SIGN_IN),
+        icon: iconNameToSvg['right from bracket'],
+      },
+    ],
+    [navigate],
+  );
+
```

Comment:
So far, I have added menu items to the App component as a mock. However, in the future, I plan to pass them from pages through PageLayout props